tests/experimental/metric_repository/test_metric_repository.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import Mock
+
+import pytest
+
+from great_expectations.experimental.metric_repository.data_store import DataStore
+from great_expectations.experimental.metric_repository.metric_repository import (
+    MetricRepository,
+)
+from great_expectations.experimental.metric_repository.metrics import MetricRun
+
+
+@pytest.fixture
+def mock_data_store():
+    return Mock(autospec=DataStore)
+
+
+@pytest.fixture
+def mock_metric_run():
+    return Mock(autospec=MetricRun)
+
+
+@pytest.mark.unit
+def test_add_metric_run(mock_data_store: DataStore, mock_metric_run: MetricRun):
```

Comment:
Also curious : do we intend to expand this testing for OSS? Or are the mercury tests sufficient?

Reply:
The tests are more extensive for the CloudDataStore which actually does the serialization and sends the request. The MetricRepository really just defers to that to do all the work so it's just a passthrough, hence the simple test just checking to make sure the CloudDataStore method is called.